docs/glossary.md

```diff
@@ -1,9 +1,9 @@
-This glossary defines a number of common terms used throughout the documentation. For the definition of framework components consult the API docs.
+This glossary defines a number of terms commonly used across the documentation. For the definitions of framework components consult the API docs.
 
-* AEA: an Autonomous Economic Agent (AEA) is a "an intelligent agent acting on an owner's behalf, with limited or no interference, and whose goal is to generate economic value to its owner". AEAs are a special type of agent. [<a href="../index">more</a>]
+* **AEA (Autonomous Economic Agent)**: An AEA is "an intelligent agent acting on an owner's behalf, with limited or no interference, and whose goal is to generate economic value to its owner". AEAs are a special type of agent. [[more]](index.md)
```

Comment:
All links need to be reverted to old format. There is a reason for this. We have a validation for links which assumes that format!

Reply:
reverted